### ISSUE TITLE
Add ingredient scaling in recipe page

### ISSUE THREAD

**Issue by minhdinh901** [cse110-fa21-group24/cse110-fa21-group24#214]
**Description**
Add arrow buttons next to the ingredients section of the recipe page to allow users to scale recipe serving sizes up or down. We should have serving sizes of 1/4, 1/2, 1, 2, 3 to keep it simple

**Possible Solutions**
- We will have to modify how ingredients are stored when retrieving recipe data from Spoonacular, look in the Spoonacular interface and Spoonacular documentation to see how we can store the ingredient amount and ingredient separately

**Issue Points**: 3
